unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PlayerInfoCardHUD/PlayerInfoCardHUDController.cs

```diff
@@ -245,6 +262,7 @@ private void SetUserProfile(UserProfile userProfile)
 
         TaskUtils.Run(async () => await AsyncSetUserProfile(userProfile)).Forget();
     }
+
     private async UniTask AsyncSetUserProfile(UserProfile userProfile)
```

Comment:
Cancellation is missing

Reply:
I think this gets out of the scope of this refactor, since i would have to support it in the profanity filtering feature. Added a TODO comment for future refactors

Reply:
Ok, let's leave it with "TODO"